### ISSUE TITLE
Uncaught TypeError: Cannot read property 'get' of null on FFW.UI.OnSystemContext

### ISSUE THREAD

**Issue by aderiabin** [smartdevicelink/sdl_hmi#397]
### Bug Report
Uncaught TypeError: Cannot read property 'get' of null on FFW.UI.OnSystemContext

##### Reproduction Steps
1. Start WebHMI
2. Click on `VRButton` button (Button for VR commands emulation)

##### Expected Behavior
No errors are observed

##### Observed Behavior
 Error is observed in browser console:
```
 FFW.UI.OnSystemContext
 UIRPC.js:2029 Uncaught TypeError: Cannot read property 'get' of null
 at d.OnSystemContext (UIRPC.js:2029)
 at d.onSystemContextChange (Controller.js:1458)
 at d.onActivate (VRPopUp.js:246)
 at j (ember-1.0.pre.min.js:14)
 at Array.<anonymous> (ember-1.0.pre.min.js:14)
 at g.flush (ember-1.0.pre.min.js:14)
 at k (ember-1.0.pre.min.js:14)
 at Object.Ember.endPropertyChanges (ember-1.0.pre.min.js:14)
 at e.flush (ember-1.0.pre.min.js:14)
 at end (ember-1.0.pre.min.js:14)
```

##### OS & Version Information
* OS/Version: Ubuntu 18.04
* HMI: https://github.com/smartdevicelink/sdl_hmi/commit/6d32f1ba10bd6d2335d755d902788fa82c6286e9

**Comment by aderiabin** [comment]
Please notice this is regression issue.

**Comment by iCollin** [comment]
@aderiabin @YarikMamykin can you confirm issue is still valid on develop? I think this may have been fixed by #403

**Comment by aderiabin** [comment]
The issue is not reproduced on current develop (2cfbe48cde1a148666284db13a28671fb59bd8ec)
